lib/use_cases/calculate_total_cost.rb

```diff
@@ -0,0 +1,38 @@
+require 'bigdecimal'
+
+class CalculateTotalCost
+  def initialize(items_gateway:)
+    @items_gateway = items_gateway
```

Comment:
As the gateway is only used for the .all method we could have ```@items = items_gateway.all``` and then delete line 9 and the arguments to the methods.